README.md

```diff
@@ -1,55 +1,18 @@
+![Publish Wiki Status](https://github.com/nimblehq/infrastructure-templates/actions/workflows/publish-wiki.yml/badge.svg)
+![Publish to NPM Status](https://github.com/nimblehq/infrastructure-templates/actions/workflows/publish.yml/badge.svg)
+
 # Nimble Infrastructure Template
 
 An infrastructure template for web applications powered by Terraform.
 
 **Supported cloud:** AWS. We are looking for contributions to implement support for GCP, Heroku, and Azure!
 
-**Supported flavors:**
-- A `Blank` flavor that generates a blank Terraform project.
-- The `Complete` flavor generates the following infrastructure:
-
-![Diagram of the Complete Infrastructure](/img/diagram_complete.svg?raw=true)
-
-## Usage
-
-The CLI can be installed globally or run directly with `npx`:
-
-```bash
-npm install -g @nimblehq/infra-template
-nimble-infra generate {project-name}
-
-# or
-
-npx @nimblehq/infra-template generate {project-name}
-```
-
-The CLI supports the following commands:
-- `generate` - to generate a new Terraform project:
-
-```bash
-nimble-infra generate {project-name}
-```
-
-- `install` - to install a new addon/module to an existing Terraform project:
-
-```bash
-nimble-infra install {addon-name} --project {project-name}
-```
-
-> [!NOTE]\
-> Terraform Cloud is supported by default. You need to set the `organization` and `workspace` in the `terraform` block of the `main.tf` file to use it.
-> If you don't want to use Terraform Cloud, you can remove the `cloud` block in the `main.tf` file.
-
-### Reference as a template
-
-> [!NOTE]\
-> This template can be used for reference to add an addon/module to an existing Terraform project
-
-
+## Documentation
+For [usage instructions](../../wiki/getting-started) and documentation, please check out the [Wiki](../../wiki).
 
 ## Contributing
 
-Check out our [contributing guidelines](/CONTRIBUTING.md) to get started.
+Check out our [contributing guidelines](../../wiki/contributing) to get started.
```

Comment:
If there is no magic with links preprocessing, we need to set the URL that we actually have:
- 🟥 https://github.com/nimblehq/infrastructure-templates/wiki/contributing
- ✅ https://github.com/nimblehq/infrastructure-templates/wiki/Contributing-guidelines

```suggestion
Check out our [contributing guidelines](../../wiki/Contributing-guidelines) to get started.
```

Reply:
nice catch. Fixed in 4c81034.
Actually, this relative link is not perfect (e.g. it will break when viewing on local https://stackoverflow.com/a/28470884/2189968) but is still better than using absolute links.

Reply:
it is also not working when viewing in branch on github, not so sure that absolute link is bad in that context (just thoughts, no need for changes)

Reply:
Yeah, it won't work when we are working on a branch. But with the relative path, it won't be affected when we rename the repository 😂

Reply:
and it still works when it is forked, no need to change the absolute URL. I don't see a perfect solution for all cases :)

Reply:
I agree, there is no perfect solution 🙂